Frontend/src/app/errorpages/error500/error500.component.html

```diff
@@ -0,0 +1,31 @@
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>500</title>
+  <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@300;400;600;700;800&display=swap" rel="stylesheet">
+</head>
+
+<body>
+<div id="error">
+
+  <div class="error-page container">
+    <div class="col-md-8 col-12 offset-md-2">
+      <img class="img-error" src="/assets/500.svg"  style="max-width: 100%; height: auto;display: block;
```

Comment:
500. svg not exist for me
![image](https://user-images.githubusercontent.com/75213424/154852912-4436d217-2c28-4693-8a86-10093990832d.png)